format/spec.md

```diff
@@ -979,18 +980,18 @@ The unified partition type looks like `Struct<field#1, field#2>`.
 
 #### Encryption Keys
 
-Encryption keys and metadata required for decrypting the manifest list files in encrypted tables.
-There are two types of entries:
-1. `key-metadata`: serialized key-metadata of the encrypted manifest list files. The key-metadata objects include encryption keys and other fields required to decrypt a file. Since these objects are sensitive, the serialized key-metadata byte arrays are encrypted by another key. The encryption is done via the integrity-preserving AES GCM cipher, using the snapshot ID as the AAD (additional authentication data) parameter. The result of the encryption is converted to a string via base64 encoding.
-2. `key`: the AES GCM key that encrypts the manifest list key-metadata. Since these keys are sensitive, they are wrapped/encrypted in a Key Management Service (KMS), using the table master key. The result of the wrapping is converted to a string via base64 encoding.
+Keys used for table encryption can be tracked in table metadata as a list named `encryption-keys`. The schema of each key is a struct with the following fields:
 
-`encryption-keys` field of table metadata is an optional list of structs with the following fields:
+| v1 | v2 |     v3     |     Field name          |   Type.              | Description |
+|----|----|------------|-------------------------|----------------------|-------------|
+|    |    | _required_ | **`key-id`**            | `string`             | ID of the encryption key |
```

Comment:
This is described as "ID of the encryption key"
Below we have "encryption-key-id"

I do think we should try to make it more clear that these are different. Maybe the second one could be key-encrypting-key-id. OR `kek-id`.

Or maybe the description is "ID for this key" or "Id for the key in `key-metadata`"

Reply:
I agree but I think this was solved by renaming `encryption-key-id` to `encrypted-by-id`.